packages/admin-next/dashboard/src/lib/client/auth.ts

```diff
@@ -14,9 +14,14 @@ async function login(token: string) {
   })
 }
 
+async function logout() {
```

Comment:
@olivermrbl let's add logout to the SDK now, and delete the `auth.ts` file. I believe after rebase it got recreated

Reply:
Ah nvm, you haven't rebased yet 😄 Well after you rebase, let's do that